File: Coding Challenge 2.js
function AlphabetSoup(str) { 
var  SortedStr;
var str2
SortedStr = str.split('');
str2 = SortedStr.sort();
str = str2


  // code goes here  
  return str; 

}
   
// keep this function call here 
console.log(AlphabetSoup(readline()));

File: Coding Challenge 2.cs
using System;

class MainClass {

  public static string AlphabetSoup(string str) {

    // code goes here
    char[] characters = str.ToCharArray();
    Array.Sort(characters);
    str = new string (characters);
    return str;

  }

  static void Main() {  
    // keep this function call here
    Console.WriteLine(AlphabetSoup(Console.ReadLine()));
  } 

}